src/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ set(BUILD_RTCORE_SHARED_LIBRARY 1)
 
 if (APPLE)
     message("Building pxcore for mac")
+    option(SUPPORT_DUKTAPE_DEBUGGER "SUPPORT_DUKTAPE_DEBUGGER" ON)
```

Comment:
Will it be an option to consider reuse DISABLE_DEBUG_MODE which already exists and enables/disables inspector/debugger support for Node instead of creating yet another option.

The problem with new option is that it doesn't check at all if the SUPPORT_DUKTAPE is ON.

Additionally, SUPPORT_DUKTAPE_DEBUGGER doesn't work seamlessly with all backends (if I'm not mistaken it will work on Linux only for glut but not e.g. for Wayland).

Reply:
Thanks. Yes, I made similar option we have for node debug enablement for duktape.Currently, it is verified with linux,osx and windows.